### PR TITLE
Deploying to websphere liberty

### PR DESCRIPTION
**Pull request by twerthi**
# About

Deploying Java applications to IBM WebSphere Liberty.

# Pre-requisites

- [X] The draft is complete and this post is ready to be reviewed.
- [X] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [X] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [X] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

File: blog/2021-05/deploying-to-websphere-liberty/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+ - DevOps
+ - Database Deployments
+ - Java
+---
+
+When evaluating software, it's easy to overlook an option because it doesn't list the specific stack you use.  However, just because it isn't listed, doesn't mean it isn't supported.  
+
+For Java-based applications, Octopus Deploy includes specific steps to deploy to Tomcat and Wildfly (JBOSS).  Those are the two most popular options but there are more web server technologies available.  
+
+Web servers such as Payara and IBM WebSphere Liberty will automatically deploy an application once it's placed in a specific folder. For those, writing a template for that specific server type isn't necessary.
```
Comment (TeganAli) [inline blog/2021-05/deploying-to-websphere-liberty/index.md:19]: @wordlee could the second sentence here be updated to the following without changing the meaning? 
Writing a template for those specific server types isn't necessary.

File: blog/2021-05/deploying-to-websphere-liberty/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+![](octopus-deploy-java-archive.png)
+
+Select the role and package you wish to deploy and scroll to the **Deployment** section.  The selected package will contain the version number of the package.  
+
+If you want the name to be more useful, fill in the `Deployed package file name`.  In my case, I wanted the environment name to be included so I used `petclinic_#{Octopus.Environment.Name}.war`
+
+![](octopus-package-name.png)
+
+As previously mentioned, IBM WebSphere Liberty will automatically deploy an application if placed in a specific folder called `dropins`.  The `dropins` folder is located directly off the server folder that we created earlier; `c:\wlp\usr\servers\petclinic\dropins`.
```
Comment (TeganAli) [inline blog/2021-05/deploying-to-websphere-liberty/index.md:81]: @wordlee should dropins be in bold? Or are back ticks correct here? (Also can you please delete the word "previously" when you do your pass?)

Comment (wordlee) [reply blog/2021-05/deploying-to-websphere-liberty/index.md:81]: Yes dropins should be bold :)

File: blog/2021-05/deploying-to-websphere-liberty/index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+The PetClinic application needs the `WEB-INF/classes/spring/datasource-config.xml` file updated with the database server, username, and password details.  To do this, we'll use the **Structured Configuration Variables** feature.  Scroll to the top of the step and click **CONFIGURE FEATURES**.  Enable the **Structured Configuration Variables** feature and click **OK**:
+
+![](octopus-structured-configuration-variables.png)
+
+Scroll to the **Structured Configuration Variables** section that was just added to the form and enter `WEB-INF/classes/spring/datasource-config.xml` for the value:
+
+![](octopus-structured-configuration-variables-value.png)
+
+With this feature configured, define the following project variables
```
Comment (wordlee) [inline blog/2021-05/deploying-to-websphere-liberty/index.md:95]:
```suggestion
With this feature configured, define the following project variables
```
```suggestion
With this feature configured, define the following project variables:
```

**Pull request merged**